src/azul/plugins/repository/tdr_anvil/__init__.py

```diff
@@ -113,7 +115,88 @@ def merge(cls, links: Iterable['Link']) -> 'Link':
 Links = set[Link]
 
 
-class Plugin(TDRPlugin):
+class BundleEntityType(Enum):
+    primary: EntityType = 'biosample'
+    supplementary: EntityType = 'file'
```

Comment:
There should be a good amount of documentation here (like a paragraph or two per enum item), describing what a bundle entity is, why we have `supplementary` and why we use `biosample` for `primary`.

We should also note that we can't just change the value of `primary` and expect everything to work after that, since most of the queries would have to be rewritten.